src/index.ts

```diff
@@ -19,6 +19,7 @@ import walletAccounts from './resources/wallet_accounts';
 import tools from './resources/tools';
 import tokens from './resources/tokens';
 import users from './resources/users';
+import request from './resources/request';
```

Comment:
Notice the deviation from the usual: name is `request` not the plural `requests`. Reason being thi s is what is in zai's path. e.g. `https://test.tnq.api.assemblypay.com/request/57ac2249245f4cbc81ea5ec665ddfc45/status`